src/shaders/circle.fragment.glsl

```diff
@@ -25,6 +29,15 @@ void main() {
 
     float opacity_t = smoothstep(0.0, antialiased_blur, extrude_length - 1.0);
 
+    float ang = atan( v_data.x, v_data.y );
+    float ang0r = (180. - (ang_end + 180.) + 720.) / 180. * 3.14;
+    float ang1r = (180. - (ang_start + 180.) + 720.) / 180. * 3.14;
+    if (ang < 0.)
+        ang += 6.28;
+    if ( !((ang > ang0r && ang < ang1r) ||
```

Comment:
Branches in shaders can be expensive, this block can be simplified with the following:

```c
#define DEG_TO_RAD(v) (v * 0.01745329251994329576)
#define PI 3.14159265358979323846

float angle_r = atan(v_data.y, -v_data.x) + PI;
float angle_start_r = DEG_TO_RAD(ang_start);
float angle_end_r = DEG_TO_RAD(ang_end);
opacity_t = mix(opacity_t, 0.0, float(angle_r < angle_start_r || angle_r > angle_end_r));
```

If we could do the conversion deg -> rad outside of the shader that would be ideal too.